### DISCUSSION TITLE
Can I add more drives to a Single Node Multi drive setup.

### DISCUSSION THREAD

**Discussion by hermit907** [minio/minio#15890] [Q&A] [answered]
I have a JBOD 60bay array with only 10 disks right now due to budget and other constraints. 
Can I start a SNMD setup right now, and then add more drives to increase storage capacity when needed in future. 
The documentation talks about expansion of multi node setups and I was unable to find anything about drives.

**Comment by klauspost** [comment, marked as answer]
No. That is currently not supported.
